packages/ffe-datepicker-react/src/datepicker/Datepicker.js

```diff
@@ -205,6 +205,21 @@ export default class Datepicker extends Component {
         }
     }
 
+    divBlurHandler(evt) {
+        if (!this._datepickerNode
+            || (evt.relatedTarget && !this._datepickerNode.contains(evt.relatedTarget))
+            || !this._datepickerNode.contains(evt.target)
+            || (this.datepickerCalendar && this.datepickerCalendar._wrapper && this.datepickerCalendar._wrapper.contains(evt.target) && evt.target.getAttribute('tabindex') !== '-1' && evt.currentTarget.getAttribute('tabindex') === '-1')
+        )
```

Comment:
Can you please explain what's happening here? It's a complex piece of code! 

Perhaps each `||` line could be pulled out into its own constant, so that it reads more like `if (isClosed || isClickOutsideDatepicker || isBlurOutsideDatepicker || {...`? It helps document the code for future developers.